src/recovery.rs

```diff
@@ -806,10 +794,10 @@ mod tests {
             is_app_limited: false,
         };
 
-        recovery.rate_on_packet_sent(&mut pkt_2, Instant::now(), TRACE_ID);
+        recovery.rate_on_packet_sent(&mut pkt_2, Instant::now());
         std::thread::sleep(Duration::new(0, 50000000));
```

Comment:
if we are changing the rate to bytes/s, lets change the sleep duration to seconds too? e.g. `Duration::new(5, 0)`

Reply:
The calculation doesn't change, just the unit, and it's `f64` so you should have mostly the same precision (if not better).

Though we should probably use `Duration::from_millis()` instead of `new()`, to make this code a bit cleaner.

Reply:
Also, 5 seconds to run each of these tests is a bit too much me thinks.